test/unit/tox.ini

```diff
@@ -4,11 +4,10 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py36, py38
+envlist = py27, py36, py38, py39
 skipsdist = True
```

Comment:
Python 3.10 is out now and is the the default on Mac ;-)

Default and default, it's what you get when installing python with homebrew